manim/mobject/graphing/probability.py

```diff
@@ -283,9 +283,9 @@ def __init__(
             (x_axis_config,), (kwargs.pop("x_axis_config", None),)
         )
 
-        self.bars = None
-        self.x_labels = None
-        self.bar_labels = None
+        self.bars: VGroup
+        self.x_labels: VGroup
```

Comment:
I find this change rather suspicious. Because it removes the default value for the attribute so that it basically doesn't exist. I would change this to `self.x: Optional[VGroup] = None` for the three attributes because else it changes functionality.

Reply:
Except for self.bars which might be better of just being `self.bars: VGroup = VGroup()` and then remove the initialization line in `_add_bars` because it's only called in `__init__` anyway